I1348 rm dqm udpolicy

ninjapapa opened the pull request:
Fixed #1348 

Instead of remove the entire user defined policy, simply made it only take DQMState as input. Just remove the df as an input. It solves the problem but keeps the flexibility.

AliTajeldin commented:
@ninjapapa is this just the first part?  Is this just removing the DF parameter from policy and not ability to create UD policy?

ninjapapa commented:
@AliTajeldin I changed my mind :-) 
No need to delete the entire UD policy, removing DF access from UD policy is good enough. I modified the issue itself.
